minari/dataset/minari_dataset.py

```diff
@@ -194,12 +140,22 @@ def __init__(
         self._additional_data_id = 0
         if episode_indices is None:
             episode_indices = np.arange(self._data.total_episodes)
+
         self._episode_indices = episode_indices
 
+        assert self._episode_indices is not None
+
+        total_steps = sum(
+            [
+                episode["total_timesteps"]
+                for episode in self._data.get_episodes(self._episode_indices.tolist())
+            ]
+        )
```

Comment:
I think it is better to use `apply` (as done in the property `total_steps`).
This because in future apply may exploit parallelism

Reply:
(and assign the value to `self.total_steps`)

Reply:
done